src/player.html

```diff
@@ -52,10 +55,10 @@ <h1>Guess the Phrase</h1>
 				<img src="assets/img/anvil.png" aria-hidden="true" class="anvil">
 			</div>
 			<div aria-hidden="true" class="podium"></div>
-			<div class="title"></div>
-			<div class="question-num">{{ curItem+1 }}</div>
-			<div class="total-questions"></div>
-			<div ng-class="{transition: inQues}" class="answer">
+			<div class="title" aria-hidden="true" role="banner" tabindex="0"  aria-label="{{focusTitleMessage}}" ></div>
```

Comment:
I'm not sold on making this focuseable. It's not an input, so it probably shouldn't be part of the tab sequence. Keeping the `aria-label` is good, but you're also making invisible to screen readers by default via `aria-hidden="true"`. The only reason it isn't hidden is because you also added a tab index: I think we should probably remove both (`tabindex` and `aria-hidden`).